.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  - init: yarn install
+
+vscode:
+  extensions:
+    - octref.vetur@0.22.6:7frR53XJ71Pa6fZpNmC2pA==
```

Comment:
May I ask how is this token generated ?

Reply:
It was directly by GitPod when I dragged and dropped the Vetur extension actually, see https://www.gitpod.io/docs/vscode-extensions/